Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -105,12 +105,13 @@ class PlaceDetailsCardViewController: UIViewController {
         setLocation(location: locationProvider?.getCurrentLocation())
         setupCardInteractions()
 
-        pageControl.numberOfPages = place.photoURLs?.count ?? 0
+        pageControl.numberOfPages = place.photoURLs.count
     }
 
     private func setLocation(location: CLLocation?) {
         if let lastTravelTimes = place.lastTravelTime {
             self.cardView.updateTravelTimesUI(travelTimes: lastTravelTimes)
+            return
```

Comment:
I don't think there should be a return here. What if the user has moved significantly since they last viewed this card. It is possible that the travel times we have cached are now out of date. That is why I left to to display the cached times and then update when the new travel times have been fetched. If I did not do it this way, then it was possible that when scrolling quickly through the cards all the eta's would be spinners.

Possibly what we need to do is add cancelling for travel times, so if a card is no longer visible but the travel times have not completed, we can cancel the request, to stop the requests overloading. I'll have a think about how we could do this and add it to the backlog